connmgr/connmanager.go

```diff
@@ -56,13 +56,10 @@ const (
 // ConnReq is the connection request to a network address. If permanent, the
 // connection will be retried on disconnection.
 type ConnReq struct {
-	// The following variables must only be used atomically.
-	//
 	// id is the unique identifier for this connection request.
-	//
+	id atomic.Uint64
 	// state is the current connection state for this connection request.
-	id    uint64
-	state uint32
+	state atomic.Uint32
```

Comment:
Probably should add a blank line above the comment for this one since they're no longer grouped.

Reply:
Fix by updating the `connmgr` commit.